src/components/ScrollShadows.tsx

```diff
@@ -73,8 +69,8 @@ export function ScrollShadows(props: ScrollShadowsProps) {
       }
       {...tid}
     >
-      {showStartShadow && <div css={startShadowStyles} />}
-      {showEndShadow && <div css={endShadowStyles} />}
+      <div css={{ ...startShadowStyles, opacity: showStartShadow ? 1 : 0 }} data-chromatic="ignore" />
+      <div css={{ ...endShadowStyles, opacity: showEndShadow ? 1 : 0 }} data-chromatic="ignore" />
```

Comment:
Is this going to ignore everything _within_ the children / wrapped by the scroll shadow as well

If so, that seems undesirable?

Reply:
Ah right, I missed that these just sit above/below the children, and don't fully wrap them.

I guess there is, right, some overlap:

![image](https://github.com/user-attachments/assets/46cda8f1-07f9-4859-beca-07b9a83d9957)

I'm wondering how the chromatic pixel diffing thing will pick that up, b/c naively I would have thought they look at "only pixels on the screen" for their diffs, but this insinuates that they can somehow map these divs back to the pixel they display?

...or maybe they just replace their content with "a big black screen" before taking the screenshot? And that's how the pixels get ignored?

:shrug: makes sense to try! Ty!

Reply:
Here's the page that explains it a bit more: https://www.chromatic.com/docs/ignoring-elements/#dimension-changes-in-ignored-elements-still-trigger-diffs

> Adding the data-chromatic="ignore" attribute or specifying a selector with the ignoreSelectors option instructs the diffing algorithm to disregard pixels within the bounding box of the ignored element. However, dimension changes of this element still trigger a change.